cmd/thermal-recorder/config.go

```diff
@@ -135,10 +115,13 @@ func ParseConfigFiles(recorderFilename, uploaderFilename, locationFileName strin
 	if err != nil {
 		return nil, err
 	}
-	return ParseConfig(buf, uploaderBuf, locationFileName)
+	locationBuf, _ := ioutil.ReadFile(locationFileName)
```

Comment:
This error shouldn't be ignored. Use `os.IsNotExist(err)` to check for the only allowable error.